test-utils/runtime-tester/src/run_test.rs

```diff
@@ -31,10 +31,10 @@ impl Scenario {
     }
 
     pub fn run(&self) -> ScenarioResult<RuntimeStats, Error> {
-        let genesis = Genesis::test(
-            self.network_config.seeds.iter().map(|x| x.parse().unwrap()).collect(),
-            1,
-        );
+        let accounts: Vec<AccountId> =
+            self.network_config.seeds.iter().map(|x| x.parse().unwrap()).collect();
+        let clients = vec![accounts[0].clone()];
```

Comment:
Is it intentional that we only take one client here?

Reply:
Yes. This is what the old code was doing (i.e. creating just one client and setting up one validator seat).

Reply:
Ah, inded, I misread the next line!